lab_2_tokenize_by_bpe/main.py

```diff
@@ -47,6 +86,25 @@ def merge_tokens(
     :param pair: a pair of tokens to be merged
     :return: dictionary in the form of <preprocessed word: number of occurrences>
     """
+    if not isinstance(word_frequencies, dict) or not isinstance(pair, tuple):
+        return None
+    dict_merged_tokens = {}
+    for i in word_frequencies:
+        if f"'{pair[0]}', '{pair[1]}'" in str(i):
```

Comment:
Too much just to check if `i` contains the pair

Reply:
fixed